File: src/main/java/pl/mjaznicki/rezerwation/hotel/model/Pokoj.java
package pl.mjaznicki.rezerwation.hotel.model;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Getter
@Setter
@NoArgsConstructor
@Entity
@Table(name = "POKOJ")
public class Pokoj {

    @Id
    @Column(name = "ID")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "NUMER_POKOJU")
    private String numerPokoju;

    @Column(name = "WIELKOSC", nullable = false)
    private int wielkosc;
}

File: angular-app/hotel-app/src/app/rezerwacje/pokoj.ts
export class Pokoj {
  id: number;
  wielkosc: number;
  numerPokoju: string;
  constructor(id: number, wielkosc: number, numerPokoju: string){
    this.id = id;
    this.numerPokoju = numerPokoju;
    this.wielkosc = wielkosc;
  }
}
